### DISCUSSION TITLE
Toggleable YAML From Linked Pages

### DISCUSSION THREAD

**Discussion by SlRvb** [mgmeyers/obsidian-kanban#6] [Ideas & Feature Requests]
I'd love for an option to allow for toggling of the YAML properties of linked pages within the Kanban markdown page.

## Something equivalent to Notions toggles:
![Notion_RsxrxQMonL](https://user-images.githubusercontent.com/54087190/115767806-5a2bb800-a35e-11eb-920a-05ce6fc71f5d.png)

## The result would look like:
![Notion_jw2ePGWmhz](https://user-images.githubusercontent.com/54087190/115767879-72033c00-a35e-11eb-93c8-12d99bd63580.png)

**Comment by DevScribblesSomeWords** [comment]
+1 to this one... It would be a game-changer for sure ٩(◕▽◕｡)۶
